### PR TITLE
Add create/delete index to graphql cql-first (and rest-using)

### PR DESCRIPTION
**Pull request by polandll**
https://github.com/stargate/docs/issues/67

### CODE REVIEW

File: modules/developers-guide/pages/graphql-using.adoc [hunk 1]
```diff
@@ -415,7 +415,7 @@ include::example$result/gql_dropTableIfExists.result[]
 // end::DropAdvTable[]
```
Comment (dougwettlaufer) [inline modules/developers-guide/pages/graphql-using.adoc:415]: Should this file be renamed "graphql.adoc"?

File: modules/developers-guide/examples/graphql/durl_deleteInex.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+curl --location --request POST 'https://8319febd-e7cf-4595-81e3-34f45d332d2a-us-east1.apps.astra.datastax.com/api/graphql-schema' \
```
Comment (dougwettlaufer) [inline modules/developers-guide/examples/graphql/durl_deleteInex.sh:1]: This file a duplicate?

Comment (polandll) [reply modules/developers-guide/examples/graphql/durl_deleteInex.sh:1]: bad copy/paste. I've deleted it.

File: modules/developers-guide/examples/graphql/curl_deleteIndex.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+curl --location --request POST 'https://8319febd-e7cf-4595-81e3-34f45d332d2a-us-east1.apps.astra.datastax.com/api/graphql-schema' \
+--header 'X-Cassandra-Token: AstraCS:txZMMdsloiGKPZhosGAnMsHY:0d54c888f49f4c5f3201a057c5e7124c30fa573a687fd16f23cb9c98167c26c8' \
```
Comment (polandll) [inline modules/developers-guide/examples/graphql/curl_deleteIndex.sh:2]: I've replaced with generic variable version.

File: modules/developers-guide/pages/graphql-using.adoc (hunk 1 above)
Comment (polandll) [reply modules/developers-guide/pages/graphql-using.adoc:415]: No. There is graphql.adoc (overview, general info), graphql-using.adoc (cql-first), and graphql-first-using.adoc (schema-first).

**Pull request merged**